.eslintrc.js

```diff
@@ -0,0 +1,312 @@
+                    "jquery-deferred-must-complete": true,
+                    "jsdoc-format": true,
+                    "jsx-boolean-value": true,
+                    "jsx-curly-spacing": [
+                        true,
+                        "never"
+                    ],
+                    "jsx-equals-spacing": [
+                        true,
+                        "never"
+                    ],
+                    "jsx-key": true,
+                    "jsx-no-bind": true,
+                    "jsx-no-lambda": true,
+                    "jsx-no-multiline-js": true,
+                    "jsx-no-string-ref": true,
+                    "jsx-self-close": true,
+                    "jsx-space-before-closing-tag": [
+                        true,
+                        "never"
+                    ],
+                    "jsx-wrap-multiline": true,
+                    "max-func-body-length": [
+                        true,
+                        150,
+                        {
+                            "ignore-parameters-to-function-regex": "describe"
+                        }
+                    ],
+                    "max-line-length": [
+                        true,
+                        120
+                    ],
+                    "mocha-avoid-only": true,
+                    "no-backbone-get-set-outside-model": true,
+                    "no-cookies": true,
+                    "no-delete-expression": true,
+                    "no-disable-auto-sanitization": true,
+                    "no-document-domain": true,
+                    "no-document-write": true,
+                    "no-dup-actions": true,
+                    "no-duplicate-imports": true,
+                    "no-duplicate-variable": true,
+                    "no-exec-script": true,
+                    "no-for-in": true,
+                    "no-function-expression": true,
+                    "no-http-string": true,
+                    "no-implicit-dependencies": [
+                        true,
+                        "dev"
+                    ],
+                    "no-inner-html": true,
+                    "no-jquery-raw-elements": true,
+                    "no-reference-import": true,
+                    "no-shadowed-variable": true,
+                    "no-string-based-set-immediate": true,
+                    "no-string-based-set-interval": true,
+                    "no-string-based-set-timeout": true,
+                    "no-trailing-whitespace": true,
+                    "no-unnecessary-local-variable": true,
+                    "no-unnecessary-override": true,
+                    "no-unused-expression": true,
+                    "no-with-statement": true,
+                    "object-literal-sort-keys": true,
+                    "one-line": [
+                        true,
+                        "check-open-brace",
+                        "check-catch",
+                        "check-else",
```

Comment:
what does check-else mean? is it that for any if there must be an else? That'd be a bad idea though, kinda want to write as much as possible without the else part. What's `one-line`?

Reply:
This is old rule from `tslint.json`

![image](https://user-images.githubusercontent.com/13611391/65481769-de873200-dec0-11e9-8977-ec97a4c9d061.png)

Ref: https://palantir.github.io/tslint/rules/one-line/